### PR TITLE
WS2-1737: Fix config update functionality in webspark_utility module

### PR DESCRIPTION
**Pull request by tbutterf**
### Description

People are having a hard time updating to WS2 2.10.2 because of the webspark_update_9014. I discovered that the problem here is that the update_config function being used assumes that the config is already in the active configs. However, this may not always be the case.

So, I have added a check to see if it exists in the active configs, and if not, it installs the config instead of updates it.

This solves the problem.

### Links

- [JIRA ticket](https://asudev.jira.com/browse/WS2-1737)

### Checklist

- [x] Solution is documented on the Jira ticket
- [x] QA steps to verify have been included on the Jira ticket
- [x] No new PHP or JS errors
- [x] No accessibility issues are introduced with this update

Note: Sections that are not applicable for this issue can be removed.

### CODE REVIEW

**Comment by mlsamuelson** [comment]
[Visit Site](https://pr-121-webspark-ci.pantheonsite.io/)

Created multidev environment [pr-121](https://dashboard.pantheon.io/sites/aec2f75f-35eb-41f8-abe7-95b292415259#pr-121) for webspark-ci.

**Review by mlsamuelson — APPROVED** [review]
Approving, after code-review. Haven't tested by running an update. Before merging, please confirm you have, @tbutterf .

**Comment by bebaps** [comment]
Merging PR. I tested on 2 different sites running `2.9.x` and was able to upgrade to `2.10.x` and was not able to replicate the issues with the `_2014` update. The update using both the current code for the upstream and this code worked as expected. I can see the history of the `pr-507` multidev on the `asu-isearch` site where this code was applied to a website experiencing this issue, and it worked. I trust those results, and this will be something we attempt to confirm further in TEST environment.

**Pull request merged**
